insights/specs/default.py

```diff
@@ -371,6 +371,7 @@ def httpd_cmd(broker):
         "/etc/pki/ovirt-vmconsole", "/etc/nova/migration", "/etc/sysconfig",
         "/etc/cloud/cloud.cfg.d", "/etc/rc.d/init.d"])
     ls_etc = simple_command("/bin/ls -lan {0}".format(' '.join(etc_and_sub_dirs)), keep_rc=True)
+    ls_etc_systemd = simple_command("/bin/ls -lanRL /etc/systemd")
```

Comment:
You also need to add the `simple_command` spec to the `insights_archive.py` as a `simple_file` to make sure the `core` could find it from the insights-archive successfully. 

Please use the `mangle` command provide by the insights-core to get the `simple_file` of this `simple_command`